mmdet/datasets/custom.py

```diff
@@ -322,3 +322,33 @@ def evaluate(self,
                 for i, num in enumerate(proposal_nums):
                     eval_results[f'AR@{num}'] = ar[i]
         return eval_results
+
+    def __repr__(self):
+        """Print the number of instance number."""
+        dataset_type = 'Test' if self.test_mode else 'Train'
+        result = f'\n{dataset_type} dataset, number of sample {len(self)}\n'
```

Comment:
change to `result = f'\n{dataset_type} dataset with number of images {len(self)}', and instance counts: \n`